backend/src/components/user.js

```diff
@@ -203,10 +205,13 @@ function mapUserFacilityObjectForFront(data) {
   return userFacilities
 }
 
-async function getUserFacilities(req, res) {
+async function getUserFacilities(req, res, onlyWithPortalAccess) {
   try {
     let userFacilities = []
-    const operation = `ofm_bceid_facilities?$expand=ofm_facility($select=accountnumber,address1_composite,name)&$filter=(statecode eq 0 and _ofm_bceid_value eq ${req.params.contactId}) and (ofm_facility/statecode eq 0)`
+    let operation = `ofm_bceid_facilities?$expand=ofm_facility($select=accountnumber,address1_composite,name)&$filter=(statecode eq 0 and _ofm_bceid_value eq ${req.params.contactId}) and (ofm_facility/statecode eq 0)`
+    if (onlyWithPortalAccess) {
+      operation = operation + ` and (ofm_portal_access eq true)`
```

Comment:
Technically this doesn't have to be a template literal since there's no variables but it's also fine as is (and consistent with previous strings).